source/retryable-writes/retryable-writes.rst

```diff
@@ -66,9 +66,23 @@ ClientSession
    name of this object MAY vary across drivers.
 
 Retryable Error
-   An error is considered retryable if it meets any of the following criteria:
+   An error is considered retryable if it has a RetryableWriteError label in
+   its top-level "errorLabels" field.
 
-   - any network exception (e.g. socket timeout or error)
+   For server versions 4.4 and newer, MongoDB will add a RetryableWriteError label to
```

Comment:
I have to say I'm finding it strange that behavior around what the driver should do wrt retryable errors is being conveyed in the "Terms" section. It seems fine to keep the first sentence here, as it falls under criteria for a retryable error, but I think the guidance around what drivers must do feels like it belongs in an "error handling" section. Thoughts?

Reply:
Perhaps we can add a new "Determining Retryable Errors" to section at the start of ["Implementing Retryable Writes"](https://github.com/mongodb/specifications/blob/master/source/retryable-writes/retryable-writes.rst#implementing-retryable-writes). Then, the "Retryable Error" term can simply state:

> A network error or server response with a particular error code that warrants retrying a write operation. See _Determining Retryable Errors_ for more information.

Reply:
SGTM

Reply:
Matt, that's a good point. I'll add a new section. Thanks!